nwcsafpps_runner/pps_posttroll_hook.py

```diff
@@ -185,10 +192,22 @@ def create_message(self, status, mda):
                 LOG.debug("Filename = %s", filename)
                 to_send['uid'] = os.path.basename(filename)
 
-        to_send['data_processing_level'] = self.level
-        to_send['format'] = self.output_format
-        to_send['status'] = status
-        station = self.station
+        # level, output_format and station are all required fields
+        for attr in ['level', 'output_format', 'station']:
+            if not attr in self.metadata:
```

Comment:
I would have writte ` if attr not in self.metadata:` but I guess both do the same

Reply:
Ah, yeah, right @TAlonglong , looks a bit awkward now that you say it!

Reply:
I was too late seeing this - merged already. Sorry! I will fix next time if I remember!